CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 CHANGELOG
 =========
 
+1.4.0
+------------------
+
+* The `maxminddb` extension now has the arginfo when using reflection.
+  PR by Remi Collet. GitHub #75.
+* The `maxminddb` extension now provides `MINFO()` function that
+  displays the extension version and the libmaxminddb version. PR by
+  Remi Collect. GitHub #74.
+* The `maxminddb` `configure` script now uses `pkg-config` when
+  available to get libmaxmindb build info. PR by Remi Collect.
+  GitHub #73.
+
```

Comment:
Remi's surname is spelled `Collet`. We got the first one right, but the second two are not. In addition, if this is the same Remi as in Remi's Repos, thanks Remi for all your contributions to the community!

Reply:
Woops! Thanks.

Reply:
Force pushed.